src/main/java/UseCase/RegisterArtistInteractor.java

```diff
@@ -16,7 +16,7 @@ public class RegisterArtistInteractor implements RegisterArtistInputBoundary{
 
     public RegisterArtistInteractor(loginOutputBoundary registerPresenter){
         this.allUsers = Database.userList.getInstance();
-        this.registerPresenter =registerPresenter;
+        this.registerPresenter = registerPresenter;
```

Comment:
Can you undo this change? I made the same one in Jessica's open PR but this will cause merge conflicts.

Reply:
lmaooo sure